Lib/fontTools/misc/bezierTools.py

```diff
@@ -957,7 +964,7 @@ def curveLineIntersections(curve, line):
         >>> len(intersections)
         3
         >>> intersections[0].pt
-        Vector((84.90010344084884, 189.87306176459828))
+        Vector((84.90010344084885, 189.87306176459828))
```

Comment:
interesting. I think I've read somewhere that the bernstein-bezier form is more "numerically stable" than the power form (can't find the reference now). Good to see it did change something (hopefully for the better)

Reply:
Well, changing back to Bernstein changed this test value back, too :)